src/main/webapp/app/core/user/user.service.ts

```diff
@@ -30,6 +30,16 @@ export class UserService {
         return this.http.put<User>(this.resourceUrl, user, { observe: 'response' });
     }
 
+    /**
+     * Call the LDAP server to update the info of a user on the server.
+     * @param user The user to be updated from the LDAP server.
+     * @return Observable<HttpResponse<User>> with the updated user as body.
+     */
+    ldapSync(login: string): Observable<HttpResponse<User>> {
+        // @ts-ignore
```

Comment:
remove ts-ignore

Reply:
Change accepted. When I remove the //@ts-ignore I got the following error message: TS2322: Type 'Observable<User>' is not assignable to type 'Observable<HttpResponse<User>>'.   Type 'User' is missing the following properties from type 'HttpResponse<User>': body, type, clone, headers, and 4 more. So I removed the HttpResponse and made only Observable<User>.